astropy/cosmology/io/tests/test_mapping.py

```diff
@@ -129,6 +129,39 @@ def test_tofrom_mapping_move_tofrom_meta(self, cosmo, to_format, from_format):
         assert got == cosmo  # (Doesn't check metadata)
         assert got.meta["mismatching"] == "will error"
 
+    def test_to_mapping_rename_disjoint(self, cosmo, to_format):
+        """Test roundtrip of ``rename`` in ``to/from_mapping()``."""
+        # rename
+        to_rename = {"name": "name"}
+        with pytest.raises(ValueError, match="disjoint"):
+            to_format("mapping", rename=to_rename)
+
+    def test_from_mapping_rename_disjoint(self, cosmo, to_format, from_format):
+        """Test roundtrip of ``rename`` in ``to/from_mapping()``."""
+        # rename
+        m = to_format("mapping")
+
+        with pytest.raises(ValueError, match="disjoint"):
+            from_format(m, format="mapping", rename={"name": "name"})
```

Comment:
Could this and the preceding test be combined to a single parametrized test?

Reply:
Maybe, in what way do you suggest?